pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -1802,7 +1802,7 @@ let
         mktplcRef = {
           publisher = "github";
           name = "copilot-chat";
-          version = "0.12.2024013003"; # latest version compatible with vscode 1.86
+          version = "0.14.2024030801"; # compatible with vscode >= 1.87
```

Comment:
How did you find the info that this was compatible with 1.87? For some reason, the extension is disabled for me and claims to only be compatible with 1.88...

Reply:
I tested it, but since I made an error with the hash, I guess I made a mistake there too...

Reply:
I just checked, and it works very well with 1.87, see:

![image](https://github.com/NixOS/nixpkgs/assets/252042/55019846-7cbf-4320-892a-46c45a28c91d)

Reply:
Looks like it's not my day today. Sorry about that.

Reply:
That's fine. Thanks for helping! I actually have another issue when I switch back to the nixpkgs commit before the update anyway. (This time the extension is marked as compatible, but it fails when starting.) But I can live without it. I feel like Microsoft doesn't do a good job at advertising which versions of extensions are compatible with which version of VS Code. They have this information somewhere, but they don't really expose it, as they just assume that everyone is installing their extensions directly within VS Code. This makes our life as packagers more difficult.

Reply:
You're right, there's a mistake with `copilot-chat`

![image](https://github.com/NixOS/nixpkgs/assets/252042/5f85151e-1f8f-45b8-9076-33096e6a3720)

I'll provide the fix tonight.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/296765